aspnetcore/fundamentals/hosting.md

```diff
@@ -159,64 +159,65 @@ Use [Configuration](configuration.md) to set configuration values to be used by
 
 ```csharp
 public static void Main(string[] args)
-   {
-     var config = new ConfigurationBuilder()
-       .AddCommandLine(args)
-       .AddJsonFile("hosting.json", optional: true)
-       .Build();
-
-     var host = new WebHostBuilder()
-       .UseConfiguration(config)
-       .UseKestrel()
-       .Configure(app =>
-       {
-         app.Run(async (context) => await context.Response.WriteAsync("Hi!"));
-       })
-     .Build();
```

Comment:
Make one more little tiny change here: Make sure that The `.Build();` is on the same level as the rest of these. It looks like it needs to go one space to the right.